src/main/server/network/server-network-model.ts

```diff
@@ -1,6 +1,6 @@
 import { NetworkMessage } from '../../shared/network/shared-network-model';
 import { IObject } from '../../shared/util/util-model';
 
-export interface ServerNetworkMessage<T extends IObject = IObject> extends NetworkMessage<T> {
+export interface ServerNetworkMessage<T = IObject> extends NetworkMessage<T> {
```

Comment:
What if it receivs an interface that extends T? Will we able to refer its properties?

Reply:
yeah, i wouldn't see why not